packages/gatsby-react-router-scroll/src/scroll-handler.tsx

```diff
@@ -73,7 +73,7 @@ export class ScrollHandler extends React.Component<
     prevProps: LocationContext | undefined
   ): void => {
     if (this.shouldUpdateScroll(prevProps, this.props)) {
-      window.scroll(0, position)
+      ;(window.scrollTo || window.scroll)(0, position)
```

Comment:
scrollTo is working on all browsers so we shouldn't do this dance. It's just wanted bytes
```suggestion
      window.scrollTo(0, position)
```